owncloudComLibrary/src/main/java/com/owncloud/android/lib/common/http/logging/LogInterceptor.kt

```diff
@@ -85,7 +85,10 @@ class LogInterceptor : Interceptor {
     private fun logHeaders(headers: Headers): Map<String, String> {
         val auxHeaders = headers.toMap().toMutableMap()
         if (auxHeaders.contains(AUTHORIZATION_HEADER)) {
-            auxHeaders[AUTHORIZATION_HEADER] = "[redacted]"
+            val authHeaderList = auxHeaders[AUTHORIZATION_HEADER]!!.split(" ")
```

Comment:
Could this value be null? I don´t know if in this case is necessary put the exclamations !!. If it is not I would write something like this:
```
            val authHeaderList = auxHeaders[AUTHORIZATION_HEADER]?.split(" ")
            val authType = authHeaderList?.get(0)
            val authInfo = if (redactAuthHeader) "[redacted]" else authHeaderList?.get(1)
```